modules/apps/commerce/commerce-product-test/src/testFunctional/macros/FrontStore.macro

```diff
@@ -319,6 +414,32 @@ definition {
 		return "${quantityConfigurationText}";
 	}
 
+	macro viewGroupedProductEntries {
```

Comment:
Check if this is a duplicate

Reply:
This is not a duplicate: 2 new macros have been added that are related to the grouped products: `CommerceProducts.addEntriesToGroupedProduct()` and `FrontStore.viewGroupedProductEntries()`.
Anyway, I'm going to move macro to the "Add macros" commit